### DISCUSSION TITLE
Problem with my public ip for the node startup

### DISCUSSION THREAD

**Discussion by swigsstaking** [filecoin-project/lotus#11549] [Build and Sync]
Hello, I have this error which prevents me from launching my node. I checked the firewall for the port in question and tested the connection and everything works. Does anyone know what I should do?

MY LOG :

déc. 27 07:51:06 sw8c-2 systemd[1]: Started Lotus Daemon.
déc. 27 07:51:07 sw8c-2 lotus[61573]: ERROR: initializing node: starting node: failed to listen on any addresses: [listen tcp4 213.221.149.152:9876: bind: cannot assign requested address]
déc. 27 07:51:07 sw8c-2 systemd[1]: lotus-daemon.service: Main process exited, code=exited, status=1/FAILURE
déc. 27 07:51:07 sw8c-2 systemd[1]: lotus-daemon.service: Failed with result 'exit-code'.
déc. 27 07:51:07 sw8c-2 systemd[1]: lotus-daemon.service: Consumed 2.119s CPU time.
déc. 27 07:51:17 sw8c-2 systemd[1]: lotus-daemon.service: Scheduled restart job, restart counter is at 98.
déc. 27 07:51:17 sw8c-2 systemd[1]: Stopped Lotus Daemon.
déc. 27 07:51:17 sw8c-2 systemd[1]: lotus-daemon.service: Consumed 2.119s CPU time.

**Comment by TippyFlitsUK** [comment]
What does the `{API]` and `[Libp2p]` section of your daemon `config.toml` look like?

**Reply by swigsstaking** [reply]
It's all good, I put 0.0.0.0 and now it works thank's.
